kuma/static/styles/base/utilities.scss

```diff
@@ -23,6 +23,7 @@ Utility classes
 
     main > & {
         padding-top: $first-content-top-padding;
+        padding-bottom: $first-content-top-padding;
```

Comment:
Nit: The naming here is a little strange as, `$first-content-top-padding` is applied to `padding-bottom`